install-git-hook.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
@kirillmakhonin 
- rename script -> install-git-secrets-hook.sh . When other (not secret scan) hooks will be added we decide how manage installation (one or several install scripts)  
- install this hook for particular repository but not whole git in OS 
- apply .gitforbidden and  gitallowed by the same way

Reply:
This script installs itself only for current repo

Reply:
Allow rules now are used in the same way